Specs/Renderer/CubeMapSpec.js

```diff
@@ -991,41 +606,21 @@ defineSuite([
             minificationFilter : TextureMinificationFilter.NEAREST_MIPMAP_LINEAR
         });
 
-        var vs = 'attribute vec4 position; void main() { gl_PointSize = 1.0; gl_Position = position; }';
         var fs =
             'uniform samplerCube u_cubeMap;' +
             'void main() { gl_FragColor = textureCube(u_cubeMap, vec3(1.0, 0.0, 0.0)); }';
 
-        sp = ShaderProgram.fromCache({
-            context : context,
-            vertexShaderSource : vs,
-            fragmentShaderSource : fs,
-            attributeLocations : {
-                position : 0
+        var uniformMap = {
+            u_cubeMap : function() {
+                return cubeMap;
             }
-        });
+        };
 
-        sp.allUniforms.u_cubeMap.value = cubeMap;
-
-        va = new VertexArray({
+        expect({
             context : context,
-            attributes : [{
-                vertexBuffer : Buffer.createVertexBuffer({
-                    context : context,
-                    typedArray : new Float32Array([0, 0, 0, 1]),
-                    usage : BufferUsage.STATIC_DRAW
-                }),
-                componentsPerAttribute : 4
-            }]
-        });
-
-        var command = new DrawCommand({
-            primitiveType : PrimitiveType.POINTS,
-            shaderProgram : sp,
-            vertexArray : va
-        });
-        command.execute(context);
-        expect(context.readPixels()).toEqual([0, 0, 255, 255]);
+            fragmentShader : fs,
+            uniformMap : uniformMap
+        }).contextToRender([0, 0, 255, 255]);
```

Comment:
Yes, beyond the scope. Update #4817 if you want, but it could be too WebGL implementation-specific to bother.